src/containers/SearchPage/components/SearchResultItem.tsx

```diff
@@ -42,10 +69,52 @@ const SearchResultItem = ({ item, type }: Props) => {
             </SafeLink>
           </CardHeading>
           <Text>{parse(item.ingress)}</Text>
+          <Text color="text.subtle" textStyle="label.small">
+            {mainContext?.breadcrumb.join(" › ")}
+            {item.contexts && item.contexts.length > 1 && (
+              <DialogRoot>
+                <DialogTrigger asChild>
+                  <StyledButton variant="link">
+                    {t("searchPage.contextModal.button", {
+                      count: item.contexts.length - 1,
+                    })}
+                  </StyledButton>
+                </DialogTrigger>
+                <DialogContent>
+                  <DialogHeader>
+                    <DialogTitle>{t("searchPage.contextModal.heading")}</DialogTitle>
+                    <DialogCloseTrigger asChild>
+                      <IconButton variant="clear">
+                        <Cross />
+                      </IconButton>
+                    </DialogCloseTrigger>
+                  </DialogHeader>
+                  <DialogBody>
+                    <ul>
+                      {item.contexts.map((context) => (
+                        <li key={context.url}>
+                          <SafeLink to={context.url}>{item.title}</SafeLink>
+                          <Text textStyle="label.small">
+                            {context.breadcrumb.join(" › ")}
+                            &nbsp;
+                            {context.isAdditional ? (
+                              <Additional aria-hidden={false} aria-label={t("resource.tooltipAdditionalTopic")} />
+                            ) : (
+                              <Core aria-hidden={false} aria-label={t("resource.tooltipCoreTopic")} />
+                            )}
+                          </Text>
```

Comment:
Tror vi med fordel kan legge en `aria-label` på `Text`-elementet som lyder sånn ca som "Brødsmulesti: Fag, Emne, underemne. Tilleggsstoff". Da slipper vi å skru av `aria-hidden`, samt at brødsmulestien blir mer forståelig for skjermleserbrukere.

Reply:
Jeg rørte ikke innholdet i dialogen, trodde vi var enige om at det tas for seg? Men jeg kan godt gjøre det i denne om det er best. Vi burde kanskje ha en aria-label på brødsmulesti som vises på kortet og ..

Reply:
Tror du fint kan røre her, faktisk. Dette er vel ikke en av de kjipe dialogene!